opal/core/search/tests/test_extract.py

```diff
@@ -370,7 +370,10 @@ def test_get_row(self):
         self.episode.set_tag_names(["trees"], self.user)
         # make sure we keep historic tags
         self.episode.set_tag_names(["leaves"], self.user)
-        self.assertIn("trees;leaves", renderer.get_row(self.episode))
+        row = renderer.get_row(self.episode)
+        self.assertTrue(
+            "trees;leaves" in row or "leaves;trees" in row
```

Comment:
Is this about postgres? 
Because order is somehow not deterministic in the query and so it comes out differently in the seralization depending on arbitrary decisions made by the RDBMS ?

Reply:
Yeah, we don't expressly order the queryset and it comes out in a different order to sqlite.

Not what I was expecting but not actually a problem. Maybe when we migrate search over, it might be worth ordering by tag name just for consistency.